src/libvirtApi/domain.ts

```diff
@@ -96,11 +96,11 @@ export const domainCanPause = (vmState: VMState) => vmState == 'running';
 export const domainCanRename = (vmState: VMState) => vmState == 'shut off';
 export const domainCanResume = (vmState: VMState) => vmState == 'paused';
 export const domainIsRunning = (vmState: VMState) => domainCanReset(vmState);
-export const domainSerialConsoleCommand = ({ vm, alias } : { vm: VM, alias: string }) => {
+export const domainSerialConsoleCommand = ({ vm, alias } : { vm: VM, alias: optString }) => {
     if (vm.displays.find(display => display.type == 'pty'))
         return ['virsh', '-c', `qemu:///${vm.connectionName}`, 'console', vm.name, alias || ''];
     else
-        return false;
+        return [];
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2267-42a6eadf-20250808-101729-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)